extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,127 @@
+| ------------- | ------- | ---------------------------------------------------------------------------------------------------------------- |
+| top           | number  | The size of the pixel-buffer in the top border of the image. Default is `0`.                                     |
+| left          | number  | The size of the pixel-buffer in the left border of the image. Default is `0`.                                    |
+| bottom        | number  | The size of the pixel-buffer in the bottom border of the image. Default is `0`.                                  |
+| right         | number  | The size of the pixel-buffer in the right border of the image. Default is `0`.                                   |
+| borderTop     | boolean | Whether or not the pixelbuffer is included images on the top border of the first tile row. Default is `true`.    |
+| borderLeft    | boolean | Whether or not the pixelbuffer is included images on the left border of the first tile column. Default is `true`. |
+| borderBottom  | boolean | Whether or not the pixelbuffer is included images on the bottom border of the last tile row. Default is `true`.  |
+| borderRight   | boolean | Whether or not the pixelbuffer is included images on the right border of the last tile column. Default is `true`.   |
+
+### Discrete Dimension Object
+
+An additional discrete dimension of that tile pyramid. The possible values of that dimension are passed as a ruleset for discrete values.
+
+| Field Name    | Type             | Description                                                                                                      |
+| ------------- | ---------------- | ---------------------------------------------------------------------------------------------------------------- |
+| start         | number\|string   | The first possible value of that particular dimension. Either a numeric or an ISO 8601 date or datetime.         |
+| stop          | number\|string   | The last possible value of that particular dimension. Either a numeric or an ISO 8601 date or datetime.          |
+| steps         | integer          | The number of steps in that dimension, including the `start` and `stop` values as borders.                       |
+
+**steps:** This is meant as a total number of steps. Where the first value is always the `start` and the last step is always the `stop` value.
+
+## Item fields
+
+| Field Name         | Type                                                         | Description                                                  |
+| ------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| assetTemplates     | Map<string, [Asset Template Object](#asset-template-object)> | An array of Asset Template objects that denote templated arguments | 
+
+### Asset Template Object
```

Comment:
This looks like it could use the same structure as an asset, but allow replacing variables in hrefs.

Reply:
So instead of re-defining it in this spec, just refer to the Asset definition? Thats also possible. In this spec it just has a different field name `href_template` to make it clear that it is meant as a template.

Reply:
Yes, I'd do so. Reasoning is that changes to assets would not get reflected here. You can already see this as the roles property is missing, which has been added with 0.9 and thus this definition is now outdated and not aligned with the current STAC Item spec any longer. You may even put the template into `href` directly, but that's probably to be discussed with more people.